artemis-console-extension/pom.xml

```diff
@@ -87,14 +87,21 @@
                 <configuration>
                     <filesets>
                         <fileset>
-                            <directory>${basedir}/${extension.path}/build</directory>
+                            <directory>${basedir}/${extension.path}/app/build</directory>
                             <includes>
                                 <include>**/*</include>
                             </includes>
                             <followSymlinks>false</followSymlinks>
                         </fileset>
                         <fileset>
-                            <directory>${basedir}/artemis-extension/node_modules</directory>
+                            <directory>${basedir}/${extension.path}/packages/artemis-console-plugin/dist</directory>
```

Comment:
...or new ones have been added.

Reply:
the "library" (bundled using `tsup`) is consumend in the "app" (bundled using `webpack`), so should be treated as _intermediate_ artifact and is used with support from yarn workspaces here: https://github.com/apache/activemq-artemis-console/blob/f1e57e1a739aefede54ca376083cd0706255585d/artemis-console-extension/artemis-extension/app/package.json#L22